tools/packaging/static-build/ovmf/build-ovmf.sh

```diff
@@ -53,16 +54,43 @@ if [ "${ovmf_build}" == "sev" ]; then
 fi
 
 info "Building ovmf"
-build -b "${build_target}" -t "${toolchain}" -a "${architecture}" -p "${ovmf_package}"
+build_cmd="build -b "${build_target}" -t "${toolchain}" -a "${architecture}" -p "${ovmf_package}""
```

Comment:
Do you have an extra `"` at the end or `""`? Or was it intentional?

Reply:
That's intentional, as it closes the first one, as in `build_cmd="..."`.  But I agree, it's not the best thing to ready, @cmaf!

Reply:
@cmaf, I've removed the extras "" and left only the ones around the full command.